### DISCUSSION TITLE
Request: AMD GPU (HIP) support?

### DISCUSSION THREAD

**Discussion by oscarbg** [mitsuba-renderer/mitsuba3#227] [Feature requests]
Hi,
would be nice to have.. must be robust enough that Blender is using it on Linux and Windows from Vega and up..
also if CUDA support is using LLVM support, LLVM also supports HIP with minimal changes to code..
thanks..

**Comment by Speierers** [comment]
Hi @oscarbg ,

It should be possible indeed and we will look into this eventually. Although this will require a fair amount of refactoring in the `drjit-core` codebase and I do not expect it to happen soon.
